src/pal/src/exception/seh-unwind.cpp

```diff
@@ -122,6 +128,11 @@ static void WinContextToUnwindCursor(CONTEXT *winContext, unw_cursor_t *cursor)
     unw_set_reg(cursor, UNW_X86_64_R13, winContext->R13);
     unw_set_reg(cursor, UNW_X86_64_R14, winContext->R14);
     unw_set_reg(cursor, UNW_X86_64_R15, winContext->R15);
+#elif defined(_X86_)
+    unw_set_reg(cursor, UNW_REG_IP, winContext->Eip);
```

Comment:
I am sorry I have not noticed this before, but we need the Rsi and Rdi here as well (and the same in the UnwindContextToWinContext below)